examples/kubernetes/cross_account_mount/README.md

```diff
@@ -0,0 +1,58 @@
+## Dynamic Provisioning
+This example shows how to create a dynamically provisioned volume created through [EFS access points](https://docs.aws.amazon.com/efs/latest/ug/efs-access-points.html) and Persistent Volume Claim (PVC) and consume it from a pod.
+
+**Note**: this example requires Kubernetes v1.17+ and driver version >= 1.2.0.
+
+### Edit [StorageClass](./specs/storageclass.yaml)
+
+```
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: efs-sc
+provisioner: efs.csi.aws.com
+mountOptions:
+  - tls
+parameters:
+  provisioningMode: efs-ap
+  fileSystemId: fs-1234abcd
+  directoryPerms: "700"
+  gidRangeStart: "1000"
+  gidRangeEnd: "2000"
+  basePath: "/dynamic_provisioning"
+  az: us-east-1
+```
+* provisioningMode - The type of volume to be provisioned by efs. Currently, only access point based provisioning is supported `efs-ap`.
+* fileSystemId - The file system under which Access Point is created.
```

Comment:
I don't think it's necessary to copy the parameter docs from READMe, they may become out of sync

Reply:
Yeah, will get rid of the parameters